models/microsoft_ads__campaign_region_report.sql

```diff
@@ -0,0 +1,70 @@
+{{ config(enabled=var('ad_reporting__microsoft_ads_enabled', True) and var('microsoft_ads__using_geographic_daily_report', False)) }}
+
+with report as (
+
+    select *
+    from {{ var('geographic_performance_daily_report') }}
+
+), 
+
+campaigns as (
+
+    select *
+    from {{ var('campaign_history') }}
+    where is_most_recent_record = True
+
+), 
+
+accounts as (
+
+    select *
+    from {{ var('account_history') }}
+    where is_most_recent_record = True
+
+), 
+
+joined as (
+
+    select
+        report.source_relation,
+        report.date_day,
```

Comment:
Thoughts on including `country` in here as well?

Reply:
Sure! Don't see the harm.